x-pack/plugins/security_solution/server/lib/types.ts

```diff
@@ -26,6 +26,7 @@ import { PinnedEvent } from './pinned_event/saved_object';
 import { Timeline } from './timeline/saved_object';
 import { TLS } from './tls';
 import { MatrixHistogram } from './matrix_histogram';
+import { SearchTypes } from './detection_engine/signals/types';
```

Comment:
Ignore this file's changes; these new EQL typings are temporarily duplicated in both the security_solution and data plugins.